src/rerank/rerank_calculator.cc

```diff
@@ -92,6 +96,23 @@ class RerankCalculator : public CalculatorBase {
                 this->sep_token = tokenizer_session->getModelConfig().at("sep_token_id").as<int64_t>();
             }
             this->pad_token = tokenizer_session->getModelConfig().at("pad_token_id").as<int64_t>();
+
+            auto it = rerank_session->getModelConfig().find("config");
+            if (it != rerank_session->getModelConfig().end()) {
+                auto it = rerank_session->getModelConfig().at("config").as<ov::AnyMap>().find("max_position_embeddings");
```

Comment:
it will be changed from config to model_info

Reply:
compare with https://github.com/openvinotoolkit/model_server/pull/2787 - it checks two possible keys

Reply:
done